open-company/open-company-web
# emoji autocompletion fixes

[trello](https://trello.com/c/e7vlhS1W/405-emoji-autocomplete)

- [x] use cljsjs emojione
- [x] run emojione in tests
- [x] make sure headline is updated in state before saving
- [x] count emojis as single character in headline count
- [x] make completion UI use emoji sprites

To Test:

run locally and verify that the things above are indeed as described.

- The counting has initial off-by-one errors which are resolved as more characters are typed.
- I also fixed an issue with spaces being counted in weird ways
